spec/features/business_show/as_a_guest_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+
+describe " a guest" do
+  it "can view see the description" do
+    user = FactoryGirl.create(:user)
+    business = FactoryGirl.create(:business)
+    allow_any_instance_of(ApplicationController).to receive(:current_user).
+      and_return(user)
+    visit business_path(business.id)
+  end
+end
```

Comment:
I'm guessing you just left this as-is because you couldn't find a good test to write?